paasta_tools/marathon_tools.py

```diff
@@ -203,7 +203,7 @@ def get_min_instances(self):
     def get_max_instances(self):
         return self.config_dict.get('max_instances', None)
 
-    def get_instances(self):
+    def get_desired_instances(self):
```

Comment:
From the ticket description it sounds like the only thing that should respect start/stop being 0 is the actual marathon json. Status, alerting, reporting, etc should be consistent and show this as "red" if we want this to be an "non-expected" condition.